app/_hub/flash/set-dynamic-upstream-host/index.md

```diff
@@ -0,0 +1,123 @@
+---
+name: Set Dynamic Upstream Host
+publisher: Flash
```

Comment:
Where does the name `Flash` come from? I'd have expected `Anup Krai` in this field based on the GitHub org

Reply:
This plugin is developed by two contributors and team name is `Flash`. So instead of giving individual team member name we gave the team name here. Please let us know if any concerns in this.